src/lib.rs

```diff
@@ -1128,6 +1124,18 @@ r"+-----------------------------------------------------------------------------
         assert_eq!(expected, table.render());
     }
 
+    #[test]
+    fn colored_data_works() {
+        let mut table = Table::new();
+        table.add_row(Row::new(vec![TableCell::new("\x1b[31ma\x1b[0m")]));
+        let expected = "╔═══╗
+║ \u{005c}u{1b}[31ma\\u{1b}[0m ║
+╚═══╝
+";
+        println!("{}", table.render());
+        assert_eq!(expected, table.render());
+    }
+
```

Comment:
Right now this test fails even though the printed output looks as expected.

```
---- test::colored_data_works stdout ----
╔═══╗
║ a ║
╚═══╝

thread 'test::colored_data_works' panicked at 'assertion failed: `(left == right)`

Diff < left / right > :
<"╔═══╗\n║ \\u{1b}[31ma\\u{1b}[0m ║\n╚═══╝\n"
>"╔═══╗\n║ \u{1b}[31ma\u{1b}[0m ║\n╚═══╝\n"
```

I can't figure out how to get the expected text to only show a single backslash, `assert_eq` is doing some funky escaping. Here is what I've tried:

1. `\u{005c}` prints `\\`
2. `\\` prints `\\`
3. `\` prints `\\`
4. `r"\"` also prints `\\`